pkg/osquery/tables/dataflattentable/tables.go

```diff
@@ -56,12 +77,12 @@ func (t *Table) generate(ctx context.Context, queryContext table.QueryContext) (
 		if q, ok := queryContext.Constraints["query"]; ok && len(q.Constraints) != 0 {
 
 			for _, constraint := range q.Constraints {
-				plistQuery := constraint.Expression
+				dataQuery := constraint.Expression
 
-				data, err := dataflatten.PlistFile(filePath,
-					append(flattenOpts, dataflatten.WithQuery(strings.Split(plistQuery, "/")))...)
+				data, err := t.dataFunc(filePath,
+					append(flattenOpts, dataflatten.WithQuery(strings.Split(dataQuery, "/")))...)
 				if err != nil {
-					fmt.Println("parse failjure")
```

Comment:
should this be sent to the logger (if available) instead of println-ed out? it feels like a 'puts-debugging' line..

Reply:
good catch.